packages/wire-cell-toolkit/package.py

```diff
@@ -218,5 +218,8 @@ def install(self, spec, prefix):
         python(*ins)
 
     def setup_run_environment(self, env):
-        env.prepend_path("WIRECELL_PATH", self.prefix.share.wirecell);
+        env.prepend_path("WIRECELL_PATH", self.prefix.share.wirecell)        
+        if 'cuda' in self.spec:
+            env.prepend_path("LD_LIBRARY_PATH", self.spec["cuda"].prefix.lib64)
```

Comment:
Setting `LD_LIBRARY_PATH` like this should not be required.

`libcudart.so` is linked with an rpath so it is found.

I believe it then locates `libcuda.so` and others via dynamic loading.

In particular `libcuda.so` is **not** provided by Spack (beyond the dummy "stub" instance).  Rather this lib is taken from the system.  On Debian, the system `libcuda.so` gets found with no special help.  

If `libcuda.so` is provided at "weird" location then `LD_LIBRARY_PATH` would be needed.  Or, likewise if the user is relocating the Spack install area then a/many `LD_LIBRARY_PATH` settings would be needed.

But in both cases the `.prefix` value would not supply the appropriate location.